pages/apis/rest_api/teams.md

```diff
@@ -57,3 +61,164 @@ Optional [query string parameters](/docs/api#query-string-parameters):
+    <tr><th><code>is_default_team</code></th><td>Whether new organization members are assigned to this team by default (<code>true</code>, <code>false</code>)</td></tr>
+    <tr><th><code>default_member_role</code></th><td>The default role assigned to members of this team (<code>member</code>, <code>maintainer</code>)</td></tr>
+    <tr><th><code>members_can_create_pipelines</code></th><td>Whether or not team members can create new pipelines (<code>true</code>, <code>false</code>)</td></tr>
+  </tbody>
+</table>
+
+Required scope: `write_teams`
+
+Success response: `201 Created`
+
+Error responses:
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>422 Unprocessable Entity</code></th><td><code>{ "message": "Validation failed: Reason for failure" }</code></td></tr>
+</tbody>
+</table>
+
+## Update a team
+
+Updates an team.
+
+```bash
+curl -H "Authorization: Bearer $TOKEN" \
+  -X PATCH "https://api.buildkite.com/v2/organizations/{org.slug}/teams/{team.uuid}" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "name": "New and Improved Backenders V2!",
+    "description": "Updated backend engineers team at Acme Inc",
+    "privacy": "visible",
+    "is_default_team": true,
+    "default_member_role": "maintainer",
+    "members_can_create_pipelines": false
+  }'
+```
+
+```json
+{
+  "name": "New and Improved Backenders V2!",
+  "description": "Updated backend engineers team at Acme Inc",
+  "privacy": "visible",
+  "is_default_team": true,
+  "default_member_role": "maintainer",
+  "members_can_create_pipelines": false
+}
+```
+
+Required [request body properties](/docs/api#request-body-properties):
+
+<table class="responsive-table">
+  <tbody>
+    <tr><th><code>name</code></th><td>Name of the team</td></tr>
+    <tr><th><code>description</code></th><td>Description of the team</td></tr>
+    <tr><th><code>privacy</code></th><td>Privacy setting of the team (<code>visible</code>, <code>secret</code>)</td></tr>
+    <tr><th><code>is_default_team</code></th><td>Whether new organization members are assigned to this team by default (<code>true</code>, <code>false</code>)</td></tr>
+    <tr><th><code>default_member_role</code></th><td>The default role assigned to members of this team (<code>member</code>, <code>maintainer</code>)</td></tr>
+    <tr><th><code>members_can_create_pipelines</code></th><td>Whether or not team members can create new pipelines (<code>true</code>, <code>false</code>)</td></tr>
+  </tbody>
+</table>
+
+Required scope: `write_teams`
+
+Success response: `200 OK`
+
+Error responses:
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>422 Unprocessable Entity</code></th><td><code>{ "message": "Validation failed: Reason for failure" }</code></td></tr>
+</tbody>
+</table>
+
+## Delete a team
+
+Remove a team.
+
+```bash
+curl -H "Authorization: Bearer $TOKEN" \
+  -X DELETE "https://api.buildkite.com/v2/organizations/{org.slug}/teams/{team.uuid}"
```

Comment:
On a few of these `DELETE` calls, I removed the `-H "Content-Type: application/json" \` options as I strongly suspect these are not required (by virtue of no JSON payload being required in the request).